app/classes/Origami/RepositoryDiscovery.php

```diff
@@ -86,9 +86,12 @@ public static function search($data, $app) {
 		}
 
 		// Remove components that have not been seen for a while
+		/** AO: Removing this deletion query while we're having problems with the Registry updates.
+				Will reinstate when things start to clear up.
 		$app->db_write->query('SET foreign_key_checks = 0;');
 		$app->db_write->query('DELETE c, cv, d, cd FROM components c LEFT JOIN componentversions cv ON c.id=cv.component_id LEFT JOIN demos d ON c.id=d.componentversion_id LEFT JOIN componentdependencies cd ON cv.id=cd.parent_version_id WHERE c.datetime_last_discovered < (NOW() - INTERVAL 3 DAY);');
```

Comment:
It explains them in the query if you follow it through `FROM components c`. Just a shorthand for mysql to make writing queries easier